orttraining/orttraining/test/training_ops/cuda/fake_quant_test.cc

```diff
@@ -79,5 +110,80 @@ TEST(CudaKernelTest, FakeQuant) {
 }
 #endif
 
+class FakeQuantGradParameterizedTest : public ::testing::TestWithParam<TensorShape> {
+};
+
+TEST_P(FakeQuantGradParameterizedTest, FakeQuantGradComputation) {
+  std::vector<std::unique_ptr<IExecutionProvider>> providers;
+  providers.emplace_back(DefaultCpuExecutionProvider());
+#ifdef USE_CUDA
```

Comment:
as ROCM has been used by some customers, shall we enable for it?

Reply:
I think I will make a separate PR for enabling QAT for rocm. Will enable bothe `FakeQuant` as well as `FakeQuantGrad` and will add unit tests for them all.